scripts/variantstore/wdl/GvsWithdrawSamples.wdl

```diff
@@ -6,14 +6,19 @@ workflow GvsWithdrawSamples {
     String dataset_name
     String project_id
 
-    Array[String] sample_names
+    String sample_name_column_name_in_file
+    File sample_names_to_include_file
+    # should be in the format "2022-01-01 00:00:00 UTC"
```

Comment:
what happens if it's not?

Reply:
Currently this would set `withdrawn` to whatever this value is. The column is a `TIMESTAMP` so if BQ can unambiguously parse the value as such it would just go in, otherwise we'd get an error. I think that's okay for now?

Reply:
BigQuery could either munge it or not accept it as a proper TIMESTAMP value